develop/components/pages/DropdownPage.tsx

```diff
@@ -32,6 +37,7 @@ const DropdownPage: React.FunctionComponent = () => {
                             selectedValue={dropDownList1Selected}
                             onChange={(value: DropdownItem) => setDropdownList1Selected(value)}
                             disabled={disabled}
+                            error={error}
```

Comment:
Putting an error message on all dropdowns can be confusing for the user, i suggest to have one dropdown with the error instead

Reply:
All the errors will be shown only if you toggle it on the bottom of the page;. Same as disabled

Reply:
@omarVodiaK can I resolve this of you want me to make changes?